lib/components/settings_manager.dart

```diff
@@ -90,35 +103,45 @@ class Prefs {
   }
 
   static int get defaultUiLanguageValue => _defaultLanguageCountry.item1.id;
+
```

Comment:
I know I told you to turn on autoformat, but my autoformat doesn't do this. Did you manually add these linebreaks?

I'm just using the default intellij formatting which I assumed was just plain dartfmt, but it's possible that it's not.

Reply:
I do have autoformatting on, I'll try to reset the settings cause there were a couple things I may have changed a while back that I don't remember.

Reply:
I'm also using intellij, I just copied over the default ide settings and it still seems to be adding line breaks between getters

Reply:
If you export your autoformatter, or commit it, I'll import it

Reply:
Just for clarity, since I'm now realizing this could come across wrong, by `I know I told you to turn on autoformat` I was trying to say 'sorry I'm complaining about the formatting after I told you to turn on the autoformatter'.

So weird. I'm pretty sure I'm using the default settings too. Will export them and check in I guess, that's a good idea.

Reply:
Yup np. It also could be that I changed the default settings somehow without remembering. Are you on Android Studio or IDEA? I'm using IDEA, that might have something to do with it.